### DISCUSSION TITLE
how can i add mermaid in my papermod

### DISCUSSION THREAD

**Discussion by xyming108** [adityatelange/hugo-PaperMod#763] [Q&A] [answered]
how can i add mermaid in my papermod

**Comment by binick** [comment]
Hi, @xyming108.
recently I added [mermaid-js](https://mermaid-js.github.io/) to my blog.

To achieve this I added [`extend_footer.html`](https://github.com/binick/binick.github.io/commit/b55365f6135be885b0201195023400948277a1bf#diff-362d2bfcb2c94aa2f74d461e907aca6c7dc38211188173591a5bde2881571218) and I have create a [`mermaid`](https://github.com/binick/binick.github.io/commit/b55365f6135be885b0201195023400948277a1bf#diff-0cd3615b3b971f656947167023fc473e3526a546c97a179936fbf754e30c4f68) shortcode.

**Reply by xyming108** [reply]
thank you, i got it

**Reply by xyming108** [reply]
😁

**Comment by xyming108** [comment, marked as answer]
这是来自QQ邮箱的自动回复邮件。您好，您的消息已收到！
